src/emcpy/stats.py

```diff
@@ -4,10 +4,12 @@
 stats.py contains statistics utility functions
 '''
 
-__all__ = ['mstats', 'lregress', 'ttest', 'get_weights', 'get_weighted_mean']
+__all__ = ['mstats', 'lregress', 'ttest', 'get_weights', 'get_weighted_mean',
+           'get_linear_regression']
 
 import numpy as _np
 from scipy.stats import t as _t
+from sklearn.linear_model import LinearRegression
```

Comment:
add whatever is required (`sklearn`) to `setup.cfg` in `install_requires` section